easybuild/easyblocks/s/scipy.py

```diff
@@ -30,34 +30,164 @@
 @author: Kenneth Hoste (Ghent University)
 @author: Pieter De Baets (Ghent University)
 @author: Jens Timmerman (Ghent University)
+@author: Jasper Grimm (University of York)
 """
+import os
+import tempfile
 from distutils.version import LooseVersion
 
-from easybuild.easyblocks.generic.fortranpythonpackage import FortranPythonPackage
-from easybuild.easyblocks.generic.pythonpackage import det_pylibdir
+import easybuild.tools.environment as env
 import easybuild.tools.toolchain as toolchain
+from easybuild.easyblocks.generic.fortranpythonpackage import FortranPythonPackage
+from easybuild.easyblocks.generic.mesonninja import MesonNinja
+from easybuild.easyblocks.generic.pythonpackage import PythonPackage, det_pylibdir
+from easybuild.framework.easyconfig import CUSTOM
+from easybuild.tools.build_log import EasyBuildError
+from easybuild.tools.filetools import change_dir, copy_dir
 
 
-class EB_scipy(FortranPythonPackage):
+class EB_scipy(FortranPythonPackage, PythonPackage, MesonNinja):
     """Support for installing the scipy Python package as part of a Python installation."""
 
+    @staticmethod
+    def extra_options():
+        """Easyconfig parameters specific to scipy."""
+        extra_vars = ({
+            'enable_slow_tests': [False, "Run scipy test suite, including tests marked as slow", CUSTOM],
+            # ignore tests by default to maintain behaviour in older easyconfigs
+            'ignore_test_result': [None, "Run scipy test suite, but ignore result (only log)", CUSTOM],
```

Comment:
help message should mention what `None` implies (and comment should also be tweaked)